emission/tests/analysisTests/TestClusterPipeline.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+import emission.core.get_database as edb
+import emission.analysis.modelling.tour_model.cluster_pipeline as cp
+import emission.simulation.trip_gen as tg
+import uuid as uu
+
+class ClusterPipelineTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        db = edb.get_trip_db()
+        if db.count() == 0:
+            tg.create_fake_trips()
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def testReadData(self):
+        db = edb.get_trip_db()
+        trips = db.find()
+        uuids = set()
+        for t in trips:
+            uuids.add(t['user_id'])
+        if len(uuids) == 0:
+            uuids.add(None)
+        self.uuids = uuids
+
+        data = cp.read_data(size=10)
+        self.assertTrue(len(data) == 10 or len(data) == 0)
```

Comment:
just in case someone runs this test without running the whole file and the database is empty.